backend/src/main/java/de/tobias/intestinalinspector/api/FoodMapDto.java

```diff
@@ -13,11 +13,19 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-public class FoodMapDto {
+public class FoodMapDto implements Comparable<FoodMapDto>{
```

Comment:
Good understanding of collections api 👍 

One might argue that this violates SOC (Separation of concerns) principle. A dtos job is to be a data holder, nothing more.

I would extract the behaviour into an `ComparatorService` that implements `Comparator` and then use `Collections.sort(someCollection, myCustomComparator)`.

See 
https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/util/Comparator.html 
and
https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/util/Collections.html#sort(java.util.List,java.util.Comparator)

Maybe research and refactor some time in the future?